src/GafferCycles/IECoreCyclesPreview/Renderer.cpp

```diff
@@ -3593,9 +3647,18 @@ class CyclesRenderer final : public IECoreScenePreview::Renderer
 
 			if( m_renderState == RENDERSTATE_RENDERING )
 			{
-				m_pause = true;
-				m_session->set_pause( m_pause );
+				m_session->set_pause( true );
+			}
+		}
+
+		IECore::DataPtr command( const IECore::InternedString name, const IECore::CompoundDataMap &parameters ) override
+		{
+			if( boost::starts_with( name.string(), "ccl:" ) || name.string().find( ":" ) == string::npos )
```

Comment:
Probably should go `cycles:` same for the unit tests.

Reply:
Good catch - fixed in https://github.com/GafferHQ/gaffer/pull/4807/commits/1e9e131b6d45819435490a865d74649a54aedddb.